scripts/linters/pylint_extensions.py

```diff
@@ -418,12 +414,14 @@ class DocstringParameterChecker(checkers.BaseChecker):
             '"%s" missing in parameter documentation',
             'missing-param-doc',
             'Please add parameter declarations for all parameters.',
-            {'old_names': [('W9003', 'missing-param-doc')]}),
+            {'old_names': [('W9003', 'old-missing-param-doc')]}),
```

Comment:
If it is not added this error is raised

```
pylint.exceptions.InvalidMessageError: Message symbol 'missing-param-doc' cannot be used for 'W9003' and 'W9015' at the same time. If you're creating an 'old_names' use 'old-missing-param-doc' as the old symbol.
```

Reply:
Ohh yes, this error makes sense, but I am not sure if we need this `old_name` or not (it looks strange to me since we have been using just the same name for two different warnings) - would leave it to @DubeySandeep to take a look, since I am not sure.
@DubeySandeep PTAL here!

Reply:
@vojtechjelinek Can we create a separate issue to discuss what needs to be done for the old_names? Feel free to resolve this thread after creating an issue! (Linter team can handle that issue!)